src/Command/MarkMigrated.php

```diff
@@ -150,4 +159,95 @@ protected function markAllMigrated($path)
         }
         $adapter->commitTransaction();
     }
+
+    /**
+     * Decides based on $input if it will try to mark more than one migration
+     *
+     * @param InputInterface $input
+     * @return boolean
+     */
+    protected function isBatchMarking(InputInterface $input)
+    {
+        $version = $input->getArgument('version');
+
+        return $this->isAllVersion($version) ||
+            $this->isSkipTo($input) ||
+            $this->isUpTo($input);
+    }
+
+    /**
+     * Decides based on $input which versions it should mark as migrated
+     *
+     * @param InputInterface $input
+     * @return array of versions that should be marked as migrated
+     */
+    protected function versionsToMark(InputInterface $input)
+    {
+        $version = $input->getArgument('version');
+        $migrations = $this->getManager()->getMigrations();
+        $versions = array_keys($migrations);
+
+        if ($this->isAllVersion($version)) {
+            return $versions;
+        }
+
+        $lengthIncrease = $this->isSkipTo($input) ? 0 : 1;
+        $index = array_search($version, $versions);
+
+        if (FALSE === $index) {
```

Comment:
I've seen the other way round to be promoted quite a bit lately as it prevents assigmnent by mistake instead of checking equality.
But personally, I still prefer the one mandated by the CS.

@pedrofs please have a look at the CS article in the book:
http://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html#comparison